packages/jest-jasmine2/src/jestExpect.ts

```diff
@@ -26,6 +26,7 @@ export default function jestExpect(config: {expand: boolean}): void {
     toThrowErrorMatchingInlineSnapshot,
     toThrowErrorMatchingSnapshot,
   });
+  // @ts-expect-error: TODO
```

Comment:
this todo is "use `@jest/expect`, right?

Reply:
Yes, I have to rework this:

https://github.com/facebook/jest/blob/adf8526f5e931a9a4ddb0b3305e224dc6ce06e69/packages/jest-jasmine2/src/jestExpect.ts#L20-L29